tools/generator/Tests/Integration-Tests/Java_Lang_Enum.cs

```diff
@@ -9,8 +9,6 @@ public class Java_Lang_Enum : BaseGeneratorTest
 		[Test]
 		public void Generated_OK ()
 		{
-			//hides inherited member `Java.Lang.Object.class_ref'
-			AllowWarnings = true;
```

Comment:
If all the warnings are gone, can we remove `AllowWarnings` from the base test class?

Reply:
We *could*, but in the future we might get additional warnings that we don't want fixed at that precise instant, so I don't see much need to remove `AllowWarnings` from the base class right now.